vendor.conf

```diff
@@ -24,7 +24,8 @@ github.com/RackSec/srslog 456df3a81436d29ba874f3590eeeee25d666f8a5
 github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-github.com/docker/libnetwork b6cb1eee1e7fc27ee05f0eb830d3e60e67a88565
+github.com/docker/libnetwork cc3847706393056f64b550385ecbfe212287408e https://github.com/aboch/libnetwork_new.git 
```

Comment:
Should we wait for a specific PR to go into libnetwork upstream before merging this PR?